src/GovUk.Education.ExploreEducationStatistics.Admin/Services/UserManagementService.cs

```diff
@@ -262,6 +277,91 @@ await _usersAndRolesDbContext.UserInvites
                 });
         }
 
+        public async Task<Either<ActionResult, Unit>> InviteContributor(string email, Guid publicationId,
```

Comment:
Discussed this method with Ben on Slack - I will pull out most of the logic from this method into something like "CreateForNewUser" and "CreateForExistingUser" to match the pattern used in PreReleaseUserService.